edm4eic.yaml

```diff
@@ -492,3 +492,14 @@ datatypes:
      - edm4eic::RawTrackerHit rawHit        // reference to the digitized hit
     OneToManyRelations:
      - edm4hep::SimTrackerHit simHits       // reference to the simulated hits
+
+  edm4eic::TrackClusterMatch:
+    Description: "Association between a Cluster and a ReconstructedParticle"
+    Author : "D. Brandenburg"
+    Members:
+      - uint32_t          cluID             // Index of corresponding cluster (position in cluster array)
+      - uint32_t          plcID             // Index of corresponding ReconstructedParticle (position in reco particle array)
+      - float             weight            // weight of this association
+    OneToOneRelations:
+      - edm4eic::Cluster  clu               // reference to the cluster
+      - edm4eic::ReconstructedParticle plc             // reference to the Reco particle
```

Comment:
I'd like to see the following changes to move forward:

1. rename clu --> cluster (`clusterID`, `cluster`), which is in-line with how we call relations to clusters in the rest of the data model 
2. rename plc --> track ('trackID`, `track`)
3. add a new entry commented-out entry below trackID that reads: `#   - int32_t trackID.    // Index of the corresponding Track (position in the track particle array)`, and a similar line in the OneToOneRelations
4. Add to the comments for the `ReconstructedParticle` association something to the extend that this is a placeholder and will be removed in favor of a link to Track once the reconstruction is properly updated.

With these changes we can move forward and merge.

Reply:
Addressed.